app/com/gu/floodgate/jobhistory/JobHistoryService.scala

```diff
@@ -1,15 +1,22 @@
 package com.gu.floodgate.jobhistory
 
+import cats.data.Validated
 import com.gu.floodgate.DynamoDBTable
-import com.gu.floodgate.contentsource.ContentSource
+import com.gu.floodgate.reindex.ReindexStatus
+import com.gu.floodgate.reindex.ReindexStatus.{ Unknown }
+import com.gu.scanamo.DynamoFormat
+import org.joda.time.DateTime
 
 import scala.concurrent.Future
 
 class JobHistoryService(jobHistoryTable: DynamoDBTable[JobHistory]) {
 
-  def createJobHistory(jobHistory: JobHistory) = jobHistoryTable.saveItem(jobHistory)
+  implicit val dateFormat = DynamoFormat.xmap(DynamoFormat.stringFormat)(d => Validated.valid(new DateTime(d)))(_.toString)
```

Comment:
Be warned that I've just realised I can probably make that first parameter implicit, so the signature is likely to change in future.